### PR TITLE
Fixes for Screen and Page Props

### PR DESCRIPTION
**Pull request by Conor-Mack**

### CODE REVIEW

File: packages/builder/src/components/userInterface/FrontendNavigatePane.svelte [hunk 1]
```diff
@@ -9,6 +9,7 @@
     newScreenPicker.show()
   }
 
+  $: console.log("STORE SCREENS", $store.screens)
```
Comment (shogunpurple) [inline packages/builder/src/components/userInterface/FrontendNavigatePane.svelte:12]: Remove log line

File: packages/builder/src/components/userInterface/ComponentPropertiesPanel.svelte [hunk 2]
```diff
@@ -30,26 +31,53 @@
...
+       componentInstance = {...$store.currentPreviewItem, ...$store.currentComponentInfo}
+     }else {
+       componentInstance = $store.currentComponentInfo
+     }
+  }
 
   const onStyleChanged = store.setComponentStyle
-  const onPropChanged = store.setComponentProp
+
+  function onPropChanged(key, value) {
```
Comment (shogunpurple) [inline packages/builder/src/components/userInterface/ComponentPropertiesPanel.svelte:72]: think you can just do

```js
if($store.currentView !== "component") {
   store.setComponentProp(key, value);
  return;
}
store.editPageOrScreen(key, value)
```

**Review by shogunpurple — APPROVED** [review]
Few minors, but once lint errors are fixed LGTM

File: packages/builder/src/components/userInterface/ComponentPropertiesPanel.svelte (hunk 2 above)
Comment (Conor-Mack) [reply packages/builder/src/components/userInterface/ComponentPropertiesPanel.svelte:72]: `$store.currentView`  seems to linger on "component" when a screen / page is selected after a component. This seems like a bit of bug so I'll dig a bit deeper.

Comment (Conor-Mack) [reply packages/builder/src/components/userInterface/ComponentPropertiesPanel.svelte:65]: @shogunpurple latest commit should do the job

**Pull request merged**